SofaKernel/modules/SofaHelper/src/sofa/helper/ComponentChange.cpp

```diff
@@ -350,10 +350,6 @@ std::map<std::string, ComponentChange> uncreatableComponents = {
     { "BeamFEMForceField", Pluginized("v20.12", "SofaGeneralSimpleFem") },
     { "HexahedralFEMForceField", Pluginized("v20.12", "SofaGeneralSimpleFem") },
     { "HexahedralFEMForceFieldAndMass", Pluginized("v20.12", "SofaGeneralSimpleFem") },
-    { "LengthContainer", Pluginized("v20.12", "SofaGeneralSimpleFem") },
-    { "PoissonContainer", Pluginized("v20.12", "SofaGeneralSimpleFem") },
-    { "RadiusContainer", Pluginized("v20.12", "SofaGeneralSimpleFem") },
-    { "StiffnessContainer", Pluginized("v20.12", "SofaGeneralSimpleFem") },
```

Comment:
To be added as "Removed" at the end of  this file?

Reply:
Nope, actually these classes were never spawnable (so a mistake to put them in the "Pluginized" list)